packages/moonstone/Picker/Picker.js

```diff
@@ -48,6 +48,16 @@ const PickerBase = kind({
 		 */
 		'aria-valuetext': PropTypes.string,
 
+		/**
+		 * By default, `data-webos-voice-labels-ext` is generated reference to children.
+		 * However, if the children is not an array of number or string, label value should be explicitly decalred.
+		 *
+		 * @type {Number[], String[]}
```

Comment:
I believe this needs to be `{Number[]|String[]}`

Reply:
I did fix it~